src/main/kotlin/io/sip3/twig/ce/configuration/SwaggerConfiguration.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2018-2020 SIP3.IO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.sip3.twig.ce.configuration
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.info.BuildProperties
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.web.bind.annotation.RestController
+import springfox.documentation.builders.ApiInfoBuilder
+import springfox.documentation.builders.PathSelectors
+import springfox.documentation.builders.RequestHandlerSelectors
+import springfox.documentation.service.Tag
+import springfox.documentation.spi.DocumentationType
+import springfox.documentation.spring.web.plugins.Docket
+import springfox.documentation.swagger2.annotations.EnableSwagger2
+
+@Configuration
+@EnableSwagger2
+open class SwaggerConfiguration {
+
+    @Autowired
+    lateinit var buildProperties: BuildProperties
+
+    @Bean
+    open fun docket(): Docket {
+        return Docket(DocumentationType.SWAGGER_2)
+                .apiInfo(ApiInfoBuilder()
+                        .title("Twig API")
+                        .version(buildProperties.version)
```

Comment:
Let's make it possible to start from IntelijIdea even when `BuildProperties` is null